docs/content/manual/_incoming/2316-2384-deploy-guest-cluster-node-selector-label copy.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Deploy guest cluster to specific node with Node selector label
+---
+
+* Related issues: [#2316](https://github.com/harvester/harvester/issues/2316) [BUG] Guest cluster nodes distributed across failure domain
+* Related issues: [#2384](https://github.com/harvester/harvester/issues/2384) [backport v1.0.3] Guest cluster nodes distributed across failure domains
+
+## Category: 
+* Rancher integration
+
+## Verification Steps
+### RKE2 Verification Steps
+1. Open Harvester Host page then edit host config 
+1. Add the following key value in the labels page:
+  - topology.kubernetes.io/zone: zone_bp
+  - topology.kubernetes.io/region: region_bp
+  ![image](https://user-images.githubusercontent.com/29251855/179735384-77e99870-92ad-41c2-b414-a872130c0b27.png)
```

Comment:
Should be indented to keep numbering consistent. I'm not sure on indentation. I use tabs which I think set to either three or 4 spaces

Reply:
Thanks for the check, I change all numbering item from space to tab indent (4 spaces), it can display the follow up numbering correct.